pkg/util/storage.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
@smarterclayton you forced me to push this out of cloud provider like 2 weeks ago and you [agreed `util.Storage` was ok](https://github.com/kubernetes/kubernetes/pull/16009#issuecomment-150346159)

I move it to `pkg/util/storage/` I'd really rather not move it again.